frontend/webapp/src/pages/index.js

```diff
@@ -17,15 +17,24 @@ export default function Home() {
     <div
       className={`${geistSans.variable} ${geistMono.variable} grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]`}
```

Comment:
@Swapnilden Please remove backed folder(including go.work.sum fule). We don't need it for this initial task